TechliftTelegramBot/Services/AgencyInfo.cs

```diff
@@ -0,0 +1,39 @@
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using Telegram.Bot.Types;
+using TechliftTelegramBot.Models;
+
+
+namespace TechliftTelegramBot.Services
+{
+    
+    public class AgencyInfo
+    {
+        private readonly HttpClient client = new();
```

Comment:
inject httpClientFactory using DI and then create a new httpClient from it whenever needed

Reply:
[check here](https://github.com/techlift-tech/TelegramBotForPokerAnalysis/commit/bda87ee1dfa450689b1b253d59be2b4e0690f348#r50255485)